src/main/java/model/ExcelImporter.java

```diff
@@ -14,7 +14,6 @@
  * A class that reads an Excel file.
  * 
  * @author Lino Silva
- * @version 1.0

```

Comment:
This didn't seem accurate, so I removed it.

Reply:
That's been pointed out in his review but I guess it passed by me in this class, thank you for removing this